packages/builder/src/pages/builder/portal/apps/index.svelte

```diff
@@ -381,8 +381,13 @@
           </div>
 
           <div class="appTable" class:unlocked>
-            {#each filteredApps as app (app.appId)}
-              <AppRow {app} {editApp} {appOverview} />
+            {#each filteredApps as app, idx}
```

Comment:
This is interesting, because by removing the key (app.appId) we're actually removing the ability for app rows to fully remount themselves when the order changes, which in my mind would actually create the sort of issues we're trying to solve here.

Reply:
I was confused by this too. My assumption was that the sorting of the filtered apps was somehow causing svelte issues when performing its diff on the rows. I thought that passing a prop that combined both the array index and the appId would solve this.

I've now altered the AppLockModal to add a `key` tag around the modal component inside. This appears to have the desired effect and the contents display without issue. I also replaced the previous key expression which, as you point out, wasn't the root of the issue.